src/manifest.rs

```diff
@@ -159,6 +159,22 @@ fn print_upgrade_if_necessary(
     Ok(())
 }
 
+#[cfg(not(feature = "pretty"))]
+fn _fmt_project(proj_header: &str, proj_data: &toml::Value, rest: &toml::Value) -> String {
+    format!("[{}]\n{}{}", proj_header, proj_data, rest)
+}
+
+#[cfg(feature = "pretty")]
+fn _fmt_project(proj_header: &str, proj_data: &toml::Value, rest: &toml::Value) -> String {
+    format!(
+        "[{}]\n{}{}",
+        proj_header,
+        toml::to_string_pretty(proj_data).expect("serde serialization"),
+        toml::to_string_pretty(rest).expect("serde serialization"),
```

Comment:
I'm not sure what "serde serialization" is supposed to tell me. The usual "expect" message format is something like "toml::Value printing shouldn't fail".

Actually, why not just return a `Result<String, Box<Error>>` here?